packages/atlas/src/components/Avatar/Avatar.tsx

```diff
@@ -38,27 +37,30 @@ export const Avatar: React.FC<AvatarProps> = ({
   size = 'default',
   children,
   onClick,
-  onEditClick,
   className,
   editable,
   newChannel,
   onError,
 }) => {
-  const handleEditClick = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
-    event.stopPropagation()
-    if (onEditClick) {
-      onEditClick(event)
-    }
-  }
   const isEditable = !loading && editable && size !== 'default' && size !== 'bid'
 
   return (
-    <Container onClick={onClick} size={size} className={className} isLoading={loading} withoutOutline={withoutOutline}>
+    <Container
+      as={onClick ? 'button' : 'div'}
+      // @ts-ignore 'type' prop will only be applied if the rendered tag is 'button' so it is valid here
```

Comment:
I'm thinking, maybe we could just override `type` in `ContainerProps` instead of commenting?

Reply:
Done